### PR TITLE
fixes for cases listed below

### PR DESCRIPTION
**Pull request by kumarabd**
Signed-off-by: kumarabd <abishekkumar92@gmail.com>

**Description**
- Bumped Meshkit version
- Exposed adapter versions to hover message on adapter chips.
- Changed graphql enums to be SMP compatible.
- Added the new revised protocol buffers as draft. This protobuf contains changes for reworked messaging system.
-  Dashboard cleanup

**Notes for Reviewers**
Dashboard Cleanup includes:
- Removing switch cases fo adapter names
- Make all the tags/names and state SMP compatible
- Istio SVG reworked to fit in

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: internal/graphql/resolver/helpers.go [hunk 1]
```diff
@@ -13,16 +13,16 @@ import (
...
-		model.MeshTypeLinkerd:     "linkerd-system",
-		model.MeshTypeConsul:      "consul-system",
-		model.MeshTypeOctarine:    "octarine-system",
-		model.MeshTypeTraefikmesh: "traefik-system",
-		model.MeshTypeOsm:         "osm-system",
-		model.MeshTypeKuma:        "kuma-system",
-		model.MeshTypeNginxsm:     "nginx-system",
-		model.MeshTypeNsm:         "nsm-system",
-		model.MeshTypeCitrix:      "ctrix-system",
+		model.MeshTypeIstio:              "istio-system",
```
Comment (leecalcote) [inline internal/graphql/resolver/helpers.go:16]: Is this list of namespaces accurate? Perhaps these value should be coming from the individual adapters.

Comment (kumarabd) [reply internal/graphql/resolver/helpers.go:16]: Yep, not accurate. Its a scenario that is considered for the time being as we discussed before.

File: helpers/adapters_tracker.go [hunk 2]
```diff
@@ -3,19 +3,23 @@ package helpers
...
 	"context"
 	"sync"
+
+	"github.com/layer5io/meshery/models"
 )
 
 // AdaptersTracker is used to hold the list of known adapters
 type AdaptersTracker struct {
-	adapters     map[string]struct{}
+	adapters     map[string]models.Adapter
```
Comment (leecalcote) [inline helpers/adapters_tracker.go:12]: Do we have a struct that represents a Meshery deployment?

Comment (kumarabd) [reply helpers/adapters_tracker.go:12]: U mean an incluster meshery instance? No I dont think we do. @leecalcote

Comment (leecalcote) [reply helpers/adapters_tracker.go:12]: Ok. I see the preferences struct the you pointed out. This is an area for rework when Environments come...

**Review by leecalcote — APPROVED** [review]

**Pull request merged**